app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftServiceImpl.java

```diff
@@ -246,6 +256,11 @@ protected void ensureDeploymentConfig(String name, DeploymentData deploymentData
         if (jaegerCollectorUri == null) {
             jaegerCollectorUri = "http://syndesis-jaeger-collector:14268/api/traces";
         }
+        Map<String, String> productizedLabels = new HashMap<>(4);
+        if (!productizedLabelsTemplate.isEmpty()) {
+            productizedLabels.putAll(productizedLabelsTemplate);
+            productizedLabels.put("com.redhat.component-name", name);
```

Comment:
If I was another dev working on this upstream am not sure I'd be comfortable with certain product label being added. Maybe if we can generalise this so that any interesting labels that have been added to the DeploymentConfig could then be handled like this. Then we don't have to call it 'productized' but, dunno, linking-labels or something...